src/gui/guiFormSpecMenu.h

```diff
@@ -40,6 +41,7 @@ class ISimpleTextureSource;
 class Client;
 class TexturePool;
 class GUIScrollContainer;
+class KeyPress;
```

Comment:
Forward declaration of KeyPress is useless as it is defined in client/keycode.h (include needed for fs_key_pendig declaration).